src/Microsoft.IdentityModel.Abstractions/EventDetails.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Details of the telemetry event.
+    /// </summary>
+    public abstract class EventDetails
+    {
+        internal Dictionary<string, DataClassification> DataClassificationMapping = new Dictionary<string, DataClassification>();
```

Comment:
@pmaytak - for an opinion about thread safety.

Reply:
@pmaytak feel free to reopen if you'd like to comment. Marking as resolved for now.